src/tcgwars/logic/impl/gen4/Arceus.groovy

```diff
@@ -248,6 +248,7 @@ public enum Arceus implements LogicCardInfo {
               if (r==PLAY_FROM_HAND && my.deck && confirm("Use Snow Gift")) {
                 powerUsed()
                 my.deck.search(min:1,"Search your deck for a card",{true}).moveTo(hidden:true,my.hand)
+                shuffleDeck()
```

Comment:
Is there ever a situation where a `deck.search` should not shuffle? I wonder if this should be added to the `search` method.

Reply:
That's a good point, I'll look for instances where there isn't a need to shuffle after a search